test/loopback_sdk_angular_test.js

```diff
@@ -51,5 +51,14 @@ exports.loopback_angular = {
     test.equal(parse.moduleName(script), 'customServices');
     test.equal(parse.baseUrl(script), 'http://custom/api');
     test.done();
+  },
+  ignore_models_option: function(test) {
+    var script = grunt.file.read('tmp/ignore_models_option');
+    test.equal(parse.moduleName(script), 'modelsToIgnoreService');
+    test.equal(parse.baseUrl(script), 'http://ignore_models/api');
```

Comment:
These two checks (module name, base url) are not relevant for this test and are already covered by `custom_options` test, please remove.